src/chrome/crdpMultiplexing/webSocketToLikeSocketProxy.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import { logger } from 'vscode-debugadapter';
+import * as WebSocket from 'ws';
+import { LikeSocket } from 'noice-json-rpc';
+
+export class WebSocketToLikeSocketProxy {
+    private _server: WebSocket.Server;
+
+    constructor(private _port: number, private _socket: LikeSocket) { }
+
+    public start(): void {
+        this._server = new WebSocket.Server({ port: this._port }, () => {
+            logger.log(`CRDP Proxy listening on: ${this._port}`);
+        });
+
+        this._server.on('connection', openedWebSocket => {
+            logger.log(`CRDP Proxy accepted a new connection`);
+            openedWebSocket.on('message', data => {
```

Comment:
The idea was to use the Multiplexor to put 1 websocket in, and get 2 websockets out, but given that the noice-json-rpc client uses LikeSockets and it seems like a simpler interface, I implemented the Multiplexor to put 1 LikeSocket in and get 2 (or n) LikeSockets out...
The reason we need the WebServer is because we need to open and listen in a port, to be able to communicate with the other process.